src/components/profile/profile.tsx

```diff
@@ -49,12 +48,12 @@ class Profile extends React.Component<Props, {}> {
         }
     }
 
-    public logout = () => {
+    public handleLogout = () => {
         this.props.logout();
         goToUrl("/");
     };
 
-    public goToTransactionDetailView = (transaction: Transaction) => {
+    public handleTransactionView = (transaction: Transaction) => {
```

Comment:
I appreciate you conforming to the `handle` aspect now. However, please change this function name to `handleTransactionViewNavigation`.